src/components/Code/Code.tsx

```diff
@@ -25,10 +25,10 @@ export const Code: FunctionComponent<CodeProps> = ({
       {(props) => (
         <Box
           as="pre"
-          border="base"
           borderRadius="md"
           boxShadow="base"
           className={props.className}
+          marginX="0.5em"
```

Comment:
I'd strongly prefer if we avoided using `em` for spacing across the board.

The size of an `em` is relative to it's parent font-size potentially leading to inconsistent spacing values. I would suggest sticking to the number system we use for spacing to remain on a consistent spacing scale [reference here](https://chakra-ui.com/docs/theming/theme#spacing)